CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
CMake 3.20 required for the presets

Reply:
I think this is aligned and serve as a example towards showing up a time-series of events.

Reply:
I meged it in my fork mm-s/DebugVision.git